client.py

```diff
@@ -8,6 +8,47 @@
 TOKEN = os.environ.get("GIT_TOKEN")
 git = Github(TOKEN)
 
+def create_md_table(project):
+    """
+    Uses 'project' parameter to generate markdown table.
+    :param project: 
+    :return: 
+    """
+    
+    current_dir = os.path.dirname(os.path.realpath(__file__))
+    json_data = open('./changelog.json').read()
```

Comment:
On this line, you are opening a file, but never close it.
Check Line 50, where you closed the md_file. 
json_data never gets closed.